forge-ai/src/main/java/forge/ai/ability/CountersPutOrRemoveAi.java

```diff
@@ -197,53 +197,51 @@ protected boolean doTriggerAINoCost(Player ai, SpellAbility sa, boolean mandator
      */
     @Override
     public CounterType chooseCounterType(List<CounterType> options, SpellAbility sa, Map<String, Object> params) {
-        if (options.size() > 1) {
```

Comment:
@tool4ever the stuff with options.size > 1 was so that when there is no option or only one, then the whole logic below doesn't need to be checked

Reply:
Yes, but the PlayerControllerAi already checks for that earlier anyway, so we don't need to check for that in every effect API again